package.json

```diff
@@ -37,8 +37,8 @@
 		"build:browsertest": "lerna run build:browsertest",
 		"cover": "lerna run cover",
 		"init": "./scripts/init.sh",
-		"precommit": "lint-staged && npm run lint",
-		"prepush": "npm run lint"
+		"postinstall": "npm run bootstrap",
+		"precommit": "lint-staged"
```

Comment:
shouldn't it have husky property with the hook setting?

Reply:
I didn't look into that but It seems husky support npm scripts too. But i will double check.